lib/helpers/getLocaleRoutesForI18n.js

```diff
@@ -15,8 +9,7 @@ const getLocaleRoutesForI18n = ({ route, srcRoute }) => {
   return i18n.locales.map((locale) => {
     return {
       route: `/${locale}${route}`,
-      dataRoute: getI18nDataRoute(route, locale),
-      nakedRoute: route,
+      dataRoute: getDataRouteForRoute(route, locale),
     };
   });
 };
```

Comment:
rn this is only used by gsp/pages.js, might be able to do without this or rename it

Reply:
nvm its not used anywhere LuL